### PR TITLE
update zerotwo.json to use new config

### PR DESCRIPTION
**Pull request by ConnorDoesDev**

### CODE REVIEW

File: domains/zerotwo.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
-  "description": "The site for ZeroTwo's docs.",
-  "repo": "https://github.com/ZeroTwoDev",
+  "description": "landing site for zerotwo, the discord bot.",
+  "repo": "https://github.com/ZeroTwoDiscord/website",
   "owner": {
-    "username": "ZeroTwoDev",
-    "email": "halil_ismail@yahoo.com"
+    "username": "Connor",
+    "email": "[redacted] send me a dm on discord instead."
```
Comment (MaskDuck) [inline domains/zerotwo.json:6]: uh left email an empty string and add an discord pls (discord = Connor!#0800, id 744603004493365330 i suppose)

File: domains/zerotwo.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
-  "description": "The site for ZeroTwo's docs.",
-  "repo": "https://github.com/ZeroTwoDev",
+  "description": "landing site for zerotwo, the discord bot.",
+  "repo": "https://github.com/ZeroTwoDiscord/website",
   "owner": {
-    "username": "ZeroTwoDev",
-    "email": "halil_ismail@yahoo.com"
+    "username": "Connor",
```
Comment (MaskDuck) [inline domains/zerotwo.json:5]: github username pls?

File: domains/zerotwo.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+    "email": "[redacted] send me a dm on discord instead."
   },
   "record": {
-    "CNAME": "zerotwodev.github.io"
+    "CNAME": "zerotwodiscord.github.io/website"
```
Comment (MaskDuck) [inline domains/zerotwo.json:9]: cname cant have slash

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by ConnorDoesDev** [comment]
your english was giving me internal bleeding, but thanks i suppose

**Comment by MaskDuck** [comment]
i dont really think im speaking english rn

**Review by MaskDuck — APPROVED** [review]
sussy

**Pull request merged**